src/main/java/teammates/ui/controller/InstructorEditInstructorFeedbackSaveAction.java

```diff
@@ -35,6 +36,17 @@ protected void verifyAccessibleForSpecificUser() {
                 instructor, session, false, Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_SESSION);
     }
 
+    @Override
+    protected void appendToStatusToAdmin(FeedbackResponseCommentAttributes feedbackResponseComment) {
```

Comment:
since the only difference in these messages to admin is their class name, why not just use `this.getClass().getName()` so that you don't need to have these repeated codes for many of the classes.

Reply:
Yes, good point. I'll make this change.